### PR TITLE
PGI improvements

### PR DESCRIPTION
**Pull request by teolemon**
PGI improvements

### CODE REVIEW

**Comment by VaiTon** [comment]
@teolemon is this ready?

**Comment by teolemon** [comment]
It is.

**Comment by teolemon** [comment]
@VaiTon yes, merge conflict solved.

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -1822,6 +1846,63 @@ ro:Băuturi spirtoase
 zh:硬酒
 wikidata:en:Q2832208
 
+<en:Alcoholic beverages
+en:German liqueur
+
+<en:German liqueur
+de:Bayerischer Blutwurz
+protected_name_file_number:en: PGI-DE-02581
+protected_name_type:en: gi
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:1855]: gi? not pgi?

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -1514,6 +1514,30 @@ nl:Duitse bieren
 ro:Beri germane, Beri Germania, Bere germană, Bere Germania, Bere germana
 country:en:Germany
 
+<en:Beers from Germany
+de:Oktoberfestbier
+protected_name_file_number:en: PGI-DE-1034
+protected_name_type:en: pgi
+origins:en: en:Germany
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:1521]: it would be best to always use canonical names:  en:germany

**Review by stephanegigandet — APPROVED** [review]

File: taxonomies/categories.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/categories.txt:1855]: image
It's not clear: Sui Generis
image

Comment (teolemon) [reply taxonomies/categories.txt:1855]: https://www.origin-gi.com/i-gi-origin-worldwide-gi-compilation-uk.html?filter=Chiemseer%20Klosterlik%C3%B6r&cc=p

**Pull request merged**